### DISCUSSION TITLE
EFB - Flex temperature calculator

### DISCUSSION THREAD

**Discussion by hanomax24** [flybywiresim/aircraft#3665] [Q&A] [answered]
Are there any plans for a flex temperature calculator?

**Comment by hotshotp** [comment, marked as answer]
yes, a little later but its coming for sure!

**Reply by hanomax24** [reply]
Sounds good

**Reply by jzpilot87** [reply]
In the meantime you can use the web-based temp calculator at http://wabpro.cz/A320/

Very useful and pretty accurate!

**Reply by hanomax24** [reply]
Thx, works fine.

**Reply by lionhell** [reply]
Site is closed... no more wabpro !
How to do now ?

**Reply by tracernz** [reply]
> 
> 
> Site is closed... no more wabpro !
> How to do now ?

Use your judgment. If it's a long runway, put a high flex temp... if it's a short runway, hot, or high, use TOGA.
